### PR TITLE
upgrade to origami version 2.0

### PR DESCRIPTION
**Pull request by JakeChampion**
This was done following the guide at https://gist.github.com/JakeChampion/f403dbc249aada43648e244801efd214

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,18 +1,36 @@
...
 		"remark-lint": "^8.0.0",
-		"remark-preset-lint-origami-component": "^1.1.3",
+		"remark-preset-lint-origami-component": "prerelease",
 		"stylelint": "^13.7.0",
 		"stylelint-config-origami-component": "^1.0.4"
 	},
 	"scripts": {
 		"start": "npx serve ./demos/local",
-		"build": "npm_config_yes=true npx \"origami-build-tools@^10\" install && npm_config_yes=true npx \"origami-build-tools@^10\" demo"
+		"build": "npm_config_yes=true npx \"origami-build-tools@prerelease\" install && npm_config_yes=true npx \"origami-build-tools@prerelease\" demo"
```
Comment (notlee) [inline package.json:31]: how come o-date is special? shall we remove these scripts?

Comment (JakeChampion) [reply package.json:31]: What do you mean by special? This is in most of our components to get branch previews working :-D

Comment (notlee) [reply package.json:31]: Oh! Got it

**Review by chee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v5.0.0 :tada:

**Pull request merged**